src/Documentation/2.0/Transactions.md

```diff
@@ -1,6 +1,92 @@
----
+__---__
 layout: page
 title: Transactions in Orleans 2.0
 ---
 
-# Placeholder for Transactions documentation
\ No newline at end of file
+# Orleans Transactions (Beta)
+__Orleans transactions are in a beta state and should not be used in production systems.__
+Orleans supports distributed transactions against persistent grain state.
+## Setup
+Orleans transactions are opt-in.  A silo must be configured to use transactions.  If it is not, any calls to transactional grains will receive an OrleansTransactionsDisabledException.
+### Transaction Manager
+Orleans transactions requires a transaction manager which is authoritative on the state of a transaction.  Currently, the transaction manager is run within the cluster and must be configured.  This can be programmatically configured on the silo host builder using UseInClusterTransactionManager and TransactionsConfiguration.
```

Comment:
Will these render as double spaces in the resulting html?

Reply:
There are some spacing issues.. I'll address